src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -4,9 +4,18 @@ import { HistoryTimeframe } from '@shapeshiftoss/types'
 import isEmpty from 'lodash/isEmpty'
 import { ReduxState } from 'state/reducer'
 
+import { marketApi } from './marketDataSlice'
+
 export const selectMarketData = (state: ReduxState) => state.marketData.byId
 
+const selectAppState = (state: ReduxState) => state
```

Comment:
we can't do this as an input to selectors - this will cause them to recompute every time *anything* inside the state changes

Reply:
The output function is already using a selector (the querySelector) so returning the state directly in the input function is no longer an anti-pattern (the inner selector still memoized the result)

But I have rewritten the selector to make the intention clearer and no longer use the selectAppState. Thank for the feedback.